src/build.js

```diff
@@ -82,6 +82,29 @@ export const build = function(statics) {
         } else if ((!buffer && char === '{') || char === '}') {
           // First `{` after opening expression `{{`.
           expr = EXPR_RAW;
+        } else if ((!buffer && char === '!')) {
```

Comment:
after this line I recommend adding something like 

`expr = EXPR_COMMENT`

`const EXPR_COMMENT = 6;` for example

since the comment is some kind of an expression it's still correct to `commit()` but just with an empty string instead of what there is in the current buffer. the buffer should be discarded